### DISCUSSION TITLE
Patience value in beam decoding

### DISCUSSION THREAD

**Discussion by glangford** [openai/whisper#154] [General]
There is an optional `patience` parameter from the command line:

```
  --patience PATIENCE   optional patience value to use in beam decoding, as in https://arxiv.org/abs/2204.05424, the default (0.0) is equivalent to not using patience (default: 0.0)
```

In the referenced arxiv paper, "Beam Decoding with Controlled Patience", the patience factors referenced there will not translate directly (pun intended) to whisper. This is because whisper sets its patience as the command line value + 1, at least according to 

https://github.com/openai/whisper/blob/520796a34c46f7e6c56a65b50d9c6e09e2637b5d/whisper/decoding.py#L283

I just wanted to point out the difference for anyone else experimenting with the option.

**Comment by jongwook** [comment]
Thanks for pointing this out! This must have been my overlook, and I should change the default / conventional to 1.0. This would be a breaking change but hopefully no one minds.
